### ISSUE TITLE
Design the Localnet documentation section

### ISSUE THREAD

**Issue by theekrystallee** [hiero-ledger/hiero-docs#602]
```[tasklist]
### Tasks
- [x] Create a "Localnet" section and add an overview under "Networks"
- [x] Create a "Local Node" page under "Localnet"
```

**Comment by Nana-EC** [comment]
Local net has a few concepts at this point - LocalNode and then Solo

Overview - Highlight why local testing and testing outside of public networks is important - no network IO, no shared resources, ability to configure products, ability to test and run custom code.
To that regard we provided
local node - local testing of a miniature optimized network. Applicable in CI
Solo - Accurate private network for advanced testing. More info coming soon

Local Node Section
- Introduce what it is, point to repo
- Steps to get running and highlight ports and IPs
- Show a quick curl, SDK and EVM tool submission to show how to utilize it locally
- Show a github action to utilize it in CI
- Show advanced configurations e.g. enabled a feature flag
- Show how to update product docker versions to run unique builds. Point to steps to build custom images for each product.

**Comment by theekrystallee** [comment]
Resolved in [0184579](https://github.com/hashgraph/hedera-docs/commit/0184579bdfb7b115930a6dab2fb7b90e13e7bd6a)
